packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -445,9 +445,14 @@ class CodePushClient {
   }
 
   /// List all release for the provided [appId].
-  Future<List<Release>> getReleases({required String appId}) async {
+  Future<List<Release>> getReleases({
+    required String appId,
+    bool sideloadableOnly = false,
+  }) async {
     final response = await _httpClient.get(
-      Uri.parse('$_v1/apps/$appId/releases'),
+      Uri.parse(
+        '$_v1/apps/$appId/releases${sideloadableOnly ? '?sideloadable=true' : ''}',
```

Comment:
rather than manually interpolating the query param it would be nice to use `.replace(query: ...)` imo.

Reply:
Ah nice, I didn't know about `.replace` – I thought we had to provide it via either the `.http` or the `.https` constructor. Thanks, and fixed